editor/plugins/script_editor_plugin.cpp

```diff
@@ -1861,13 +1861,11 @@ void ScriptEditor::_members_overview_selected(int p_idx) {
 	if (!se) {
 		return;
 	}
-	// Go to the member's line and reset the cursor column. We can't change scroll_position
-	// directly until we have gone to the line first, since code might be folded.
-	se->goto_line(members_overview->get_item_metadata(p_idx));
-	Dictionary state = se->get_edit_state();
-	state["column"] = 0;
-	state["scroll_position"] = members_overview->get_item_metadata(p_idx);
-	se->set_edit_state(state);
+	CodeTextEditor *cte = se->get_code_editor();
```

Comment:
Can't say I like the idea of the `ScriptEditorPlugin` interacting with the code editor directly, would prefer changing the API (combine them with a new arg?) or as above cast to the specific type.